src/pages/new-vehicle.vue

```
@@ -25,72 +58,205 @@
+        uuid: user.uuid,
+        firstName: user.firstName,
+        lastName: user.lastName,
+        email: user.email,
+        oib: user.oib,
+        residence: user.residence,
+        role: user.role as UserRole,
+        birthDate: user.birthDate
+      };
+      snackbar.Success("User found successfully");
+    }
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)